php/yii/auth-bypass-yii-framework-apps3c.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Yii;
+
+class Module extends \yii\base\Module
+{
+
+    # Wrong   
```

Comment:
Can use `// ruleid:` above lines you expect to match and `// ok:` above lines you expect to not match (correct lines) -- we usually have at least one `//ruleid` and one `//ok` in every test file!